### PR TITLE
Implement cram chunking for PacBio and Nanopore

### PR DESCRIPTION
**Pull request by cduong11**
- Add `MINIMAP2_MAPREDUCE` into `align_ont` and `align_pacbio`
- Replace `seqtk/subseq` by `bbmap/filterbyread`
- Adjust resource for `GENERATE_CRAM_CSV` and `SAMTOOLS_ADDREPLACERG`

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/readmapping/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by reichan1998** [comment]
Hi @tkchafin,
I've added some updates from patch 1.3.1 (replicated your code). Please let me know if I did anything incorrectly.

File: conf/modules.config
Comment (tkchafin) [inline conf/modules.config:1]: Can we delete the modules.config entries for the older aligner calls?

File: seq_cache_populate.pl
Comment (tkchafin) [inline seq_cache_populate.pl:1]: can delete

**Review by tkchafin — CHANGES_REQUESTED** [review]
Overall looks good and I'm able to run the full pipeline with chunking across all read types now. We can track merging the PacBio filtering into the map-reduce as a separate issue. One thing it it would be nice to track down the sources of all the `WARN: The operator `first` is useless when applied to a value channel which returns a single value by definition` warnings

**Comment by reichan1998** [comment]
Hi @tkchafin, I’ve made the changes based on your feedback. Thank you!

**Review by tkchafin — APPROVED** [review]

**Pull request merged**